pyteal/compiler/compiler.py

```diff
@@ -25,17 +25,25 @@
 )
 from pyteal.compiler.constants import createConstantBlocks
 
-MAX_TEAL_VERSION = 8
-MIN_TEAL_VERSION = 2
-DEFAULT_TEAL_VERSION = MIN_TEAL_VERSION
+MAX_PROGRAM_VERSION = 7
```

Comment:
@ahangsu Can you double check me?  I think the merge conflict here requires a change.

```suggestion
MAX_PROGRAM_VERSION = 8
```

Reply:
👍 I will change accordingly